packages/outline-view/src/browser/outline-view-manager.ts

```diff
@@ -0,0 +1,52 @@
+/*
+* Copyright (C) 2017 TypeFox and others.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+import { injectable } from "inversify";
+import { Event, Emitter } from "@theia/core";
+import { ICompositeTreeNode, ISelectableTreeNode, IExpandableTreeNode, ITreeNode } from "@theia/core/lib/browser";
+
+@injectable()
+export class OutlineViewManager {
```

Comment:
I would call it `OutlineViewService`, it does not seem to manage anything. I would think that responsibilities of `OutlineViewManager` is to create instances of `OutlineView`, provide access to them and so on

Reply:
+1